packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -284,21 +286,26 @@ class ObjectInspector extends Component {
       nodeHasProperties(item)
       || nodeHasAccessors(item)
       || nodeIsMapEntry(item)
+      || nodeIsLongString(item)
       || isPrimitive
     ) {
-      let repsProp = {...this.props};
+      let repProps = {...this.props};
       if (depth > 0) {
-        repsProp.mode = this.props.mode === MODE.LONG
+        repProps.mode = this.props.mode === MODE.LONG
           ? MODE.SHORT
           : MODE.TINY;
       }
       if (expanded) {
-        repsProp.mode = MODE.TINY;
+        repProps.mode = MODE.TINY;
       }
 
+      repProps.member = {
+        open: nodeHasFullText(item) && expanded
+      };
+
       return {
         label,
-        value: Utils.renderRep(item, repsProp)
+        value: Utils.renderRep(item, repProps)
```

Comment:
Should we also pass a small `cropLimit` so that the `initial` string can be displayed in a single line, to mirror JSONViewer's way of displaying it?

Reply:
Not sure about that.

Reply:
👍